packages/gatsby-link/src/rewrite-link-path.ts

```diff
@@ -1,23 +1,28 @@
 import { resolve } from "@gatsbyjs/reach-router"
 // Specific import to treeshake Node.js stuff
-import { applyTrailingSlashOption } from "gatsby-page-utils/apply-trailing-slash-option"
+import { applyTrailingSlashOption, type TrailingSlash } from "gatsby-page-utils"
```

Comment:
The import from `gatsby-page-utils/apply-trailing-slash-option` was on purpose for better tree-shaking. So if you need the type from the root, add an additional import instead:

```ts
import type { TrailingSlash } from "gatsby-page-utils"
```